src/main/java/me/totalfreedom/totalfreedommod/rank/Title.java

```diff
@@ -7,6 +7,7 @@
 {
 
     DEVELOPER("a", "Developer", ChatColor.DARK_PURPLE, "Dev"),
+    FOUNDER("the", "Founder", ChatColor.DARK_RED, "TF Founder"),
```

Comment:
remember, this is a mod that not only TF uses. might as well just do what Ryan said...